src/C1T2X_OBU.py

```diff
@@ -39,17 +40,16 @@
 c1t2x_logger_formatter = logging.Formatter("[%(asctime)s.%(msecs)03d] %(levelname)s - %(message)s", datefmt= "%d-%b-%y %H:%M:%S")
 c1t2x_logger_handler.setFormatter(c1t2x_logger_formatter)
 c1t2x_logger.addHandler(c1t2x_logger_handler)
-# start logging
+# Start logging
 c1t2x_logger.info("\n---------------------------\nStarting C1T2X OBU Logger\n---------------------------")
 
-
-# initialize errors
+# Initialize error
 error = False
 
-# determine printing
+# Default to printing output
 printData = True
```

Comment:
This is executed as a script, so we should make this an argument with `argparse`

Reply:
This script should be running at startup as an embedded service so user input isn't expected